### PR TITLE
docs(rxAuth): FRMW-163-Migrate rxAuth to API Docs

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-163

- [x] Dev
- [x] Dev

Migrate `rxAuth` to API Documentation

### CODE REVIEW

**Comment by bry4n** [comment]
LGTM

File: src/rxAuth/README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 [![unstable](http://badges.github.io/stability-badges/dist/unstable.svg)](http://github.com/badges/stability-badges)
 
-Auth service which provides a common mechanism authenticating, validating permissions and managing sessions.
\ No newline at end of file
+`rxAuth` provides authentication services through the `Auth` service, including, common mechanism 
+authenticating, validating permissions and managing sessions.
```
Comment (CITguy) [inline src/rxAuth/README.md]: I'd go with...

>  The rxAuth component provides logic for authenticating, validating permissions, and managing sessions.

File: src/rxAuth/rxAuth.js [hunk 2]
```diff
@@ -4,8 +4,9 @@
  * @description
  * # rxAuth Component
  *
- * [TBD]
- *
+ * `rxAuth` provides authentication services through the `Auth` service, including, common mechanism 
+ * authenticating, validating permissions and managing sessions.
+ *
```
Comment (CITguy) [inline src/rxAuth/rxAuth.js]: Same description as above.

**Comment by 100stacks** [comment]
Okay made the changes.

**Comment by CITguy** [comment]
@100stacks, please squash this PR

**Comment by 100stacks** [comment]
@CITguy okay just squashed it.  Thanks.

**Pull request merged**